components/view/index.php

```diff
@@ -49,11 +59,15 @@
     });
 
     $viewResolver->register('php', function () {
-        return new PhpEngine;
+        return new PhpEngine(new Filesystem());
     });
```

Comment:
Can do this instead
```
$viewResolver->register('php', function () use ($filesystem) {
    return new PhpEngine($filesystem);
});
```

Reply:
I will bind it to the container properly as it's done in a real Laravel app.
But yes, good catch - thanks! 😊